dcbr-api/api/models.py

```diff
@@ -345,7 +345,7 @@ class Renewal(models.Model):
         max_length=20, default="", blank=True
     )
     # Note - expiry date is simply a self-reported expiry date
-    expiry_date = models.CharField(max_length=10, default= "", blank = True)
+    expiry_date = models.CharField(max_length=10, default="", blank=True)
```

Comment:
Tghis is a field that will be visually reconciled by the AGRI back-office, right?

Reply:
yes, and this is going to be changed to registration date instead of expiry.  I have an outstanding ticket to do that